index.js

```diff
@@ -316,8 +318,18 @@ module.exports = {
 
     // We have underscores in our public API already
     "camelcase": "off",
-    "capitalized-comments": ["error", "always", { ignoreConsecutiveComments: true }],
-    "comma-dangle": ["error", "always-multiline"],
+    "capitalized-comments": ["error", "always", {
+      ignorePattern: "c8",
+      ignoreConsecutiveComments: true,
+    }],
+
+    "comma-dangle": ["error", {
```

Comment:
This was overridden in peggy's .eslint before.  Now moved here where it belongs.